src/utils/comment.util.js

```diff
@@ -19,4 +19,17 @@ export default class CommentUtils {
       return Response.InternalServerError(res, 'error occured while looking for comment owner');
     }
   }
+
+  /**
+   * @param {string} requestId the id of the travel request
+   * @returns {string} the id of the owner of the request
+   */
+  static async findRequestOwnerId(requestId) {
+    try {
+      const result = await Request.findOne({ where: { id: requestId } });
+      return result;
+    } catch (error) {
+      return Response.InternalServerError(res, 'error occured while looking for request owner');
```

Comment:
Just like the first one, it is not a very good idea to leak app's vulnerabilities. Send a generic message instead. 
Or, better still, you could use the app's default 500 handler like so

```

try {
...your logic
} catch (error) { next(error); }

```